connectivity/check/deployment.go

```diff
@@ -766,9 +767,11 @@ func (ct *ConnectivityTest) deploymentList() (srcList []string, dstList []string
 	if !ct.params.Perf {
 		srcList = []string{clientDeploymentName, client2DeploymentName, echoSameNodeDeploymentName}
 	} else {
-		perfDeploymentNameManager := newPerfDeploymentNameManager(&ct.params)
-		srcList = []string{perfDeploymentNameManager.ClientName()}
-		dstList = append(dstList, perfDeploymentNameManager.ServerName())
```

Comment:
Now that `dstList` is returned empty for performance tests, we should probably skip the `ct.waitForDeployments` call on empty `dstDeployments` here:

https://github.com/cilium/cilium-cli/blob/0b7737bce14fbc7ace66aefc9973013b19ed4e64/connectivity/check/deployment.go#L823

to avoid logging a message with an empty liste of deployments:

https://github.com/cilium/cilium-cli/blob/0b7737bce14fbc7ace66aefc9973013b19ed4e64/connectivity/check/deployment.go#L1153